yarn.lock

```diff
@@ -8955,10 +8955,10 @@ yallist@^4.0.0:
   resolved "https://registry.npmjs.org/yallist/-/yallist-4.0.0.tgz"
   integrity sha512-3wdGidZyq5PB084XLES5TpOSRA3wjXAlIWMhum2kRcv/41Sn2emQ0dycQW4uZXLejwKvg6EsvbdlVL+FYEct7A==
 
-yaml@^1.10.0:
-  version "1.10.2"
-  resolved "https://registry.npmjs.org/yaml/-/yaml-1.10.2.tgz"
-  integrity sha512-r3vXyErRCYJ7wg28yvBY5VSoAF8ZvlcW9/BwUzEtUsjvX/DKs24dIkuwjtuprwJJHsbyUbLApepYTR1BN4uHrg==
+yaml@2.2.2, yaml@^1.10.0:
+  version "2.2.2"
+  resolved "https://registry.yarnpkg.com/yaml/-/yaml-2.2.2.tgz#ec551ef37326e6d42872dad1970300f8eb83a073"
```

Comment:
Not sure why registry changed but you can try setting registry locally to https://registry.npmjs.org/ and update yarn.lock file again.

Reply:
+1

Reply:
Good catch! That's a question I had for this package. The package has a `yarn.lock` so I used `yarn build`, but in the `package.json`, all scripts use `npm` 🤷

Reply:
Maybe we should standardize on yarn?

Reply:
Yeah.
For now, can you try
```
yarn config set registry https://registry.npmjs.org
yarn
```

Reply:
Looks like we use a mix of both - npm for builds, and yarn for publish. Not sure why.

Reply:
NVM, I just changed the registery to `npm` to make the PR simple